### PR TITLE
Use go-pkgs and remove dependency to iotex-core

### PR DESCRIPTION
**Pull request by dustinxie**
1. as title
2. also add make file to make fmt and make test 
3. remove dep ensure file, added go mod file

Test plan:
make test

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -27,23 +27,8 @@
...
-
-[[constraint]]
-  name = "github.com/stretchr/testify"
-  version = "1.3.0"
-
-[[constraint]]
-  name = "go.uber.org/zap"
-  version = "1.9.1"
+  revision = "2bbee71fbe61def241832e800b4f0bd24d4beb89"
+  source = "github.com/iotexproject/go-ethereum"
```
Comment (dustinxie) [inline Gopkg.toml]: same as Gopkg.toml in iotex-core

File: example.go [hunk 2]
```diff
@@ -1,18 +0,0 @@
-package main
```
Comment (zjshen14) [inline example.go]: Move this to example package

File: address/address_v1.go [hunk 3]
```diff
@@ -70,12 +69,12 @@ func (addr *AddrV1) String() string {
 	// Group the payload into 5 bit groups.
 	grouped, err := bech32.ConvertBits(payload, 8, 5, true)
 	if err != nil {
-		log.L().Panic("Error when grouping the payload into 5 bit groups.", zap.Error(err))
+		panic("Error when grouping the payload into 5 bit groups." + err.Error())
```
Comment (zjshen14) [inline address/address_v1.go]: why not log.L().Panic?

File: address/address_v1.go [hunk 4]
```diff
@@ -8,11 +8,10 @@ package address
 
 import (
 	"github.com/pkg/errors"
-	"go.uber.org/zap"
+
```
Comment (zjshen14) [inline address/address_v1.go]: Can group together?

File: address/address_v1_test.go [hunk 5]
```diff
@@ -14,12 +14,12 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
```
Comment (zjshen14) [inline address/address_v1_test.go]: Ditto

File: address/address_v1.go (hunk 4 above)
Comment (dustinxie) [reply address/address_v1.go]: fixed

File: address/address_v1_test.go (hunk 5 above)
Comment (dustinxie) [reply address/address_v1_test.go]: fixed

File: address/address_v1.go (hunk 3 above)
Comment (dustinxie) [reply address/address_v1.go]: so we don't need to import uber/zap package
feel it's like killing chicken using butcher knife :)

changed to log.Panic()

**Pull request merged**